cpp/ql/src/semmle/code/cpp/models/implementations/Strset.qll

```diff
@@ -40,6 +40,9 @@ private class StrsetFunction extends ArrayFunction, DataFlowFunction, AliasFunct
     // flow from the input string to the output string
     input.isParameter(0) and
     output.isReturnValue()
+    or
+    input.isParameterDeref(0) and
+    output.isReturnValueDeref()
```

Comment:
I don't think this is true.  If you pass in a pointer to a string, say "BADSTRING", you get out the same pointer but to a different string, say "#########".

Reply:
Fixed in 55615586eed1a4162aeff56e2d0b0bb53e64e0c9.